src/index.js

```diff
@@ -15,23 +16,23 @@ class App extends Component {
   state = {
     ocean: undefined,
     results: [],
-    ddo: undefined
+    ddo: undefined    
   }
 
   async componentDidMount() {
     const ocean = await new Ocean.getInstance({
       web3Provider: web3,
-      nodeUri: 'https://nile.dev-ocean.com',
-      aquariusUri: 'https://aquarius.marketplace.dev-ocean.com',
-      brizoUri: 'https://brizo.marketplace.dev-ocean.com',
-      brizoAddress: '0x4aaab179035dc57b35e2ce066919048686f82972',
-      secretStoreUri: 'https://secret-store.nile.dev-ocean.com',
+      //nodeUri: 'https://nile.dev-ocean.com',
+      //aquariusUri: 'https://aquarius.marketplace.dev-ocean.com',
+      //brizoUri: 'https://brizo.marketplace.dev-ocean.com',
+      //brizoAddress: '0x4aaab179035dc57b35e2ce066919048686f82972',
+      //secretStoreUri: 'https://secret-store.nile.dev-ocean.com',
       // local Spree connection
-      // nodeUri: 'http://localhost:8545',
-      // aquariusUri: 'http://aquarius:5000',
-      // brizoUri: 'http://localhost:8030',
-      // brizoAddress: '0x00bd138abd70e2f00903268f3db08f2d25677c9e',
-      // secretStoreUri: 'http://localhost:12001',
+      nodeUri: 'http://localhost:8545',
```

Comment:
switching to Spree by default is not an option cause it would break CodeSandbox, and would increase the first start difficulty of the actual tutorial: https://docs.oceanprotocol.com/tutorials/react-setup/

Reply:
bump, this needs to be addressed

Reply:
fixed. We will keep spree config commented, so it can be enabled quickly by a developer